spec/rails_helper.rb

```diff
@@ -26,6 +26,8 @@
 # If you are not using ActiveRecord, you can remove this line.
 ActiveRecord::Migration.maintain_test_schema!
 
+File.delete(Rails.root.join('config', 'system_uuid')) if File.exist?(Rails.root.join('config', 'system_uuid'))
```

Comment:
I don't think it is a good idea to manipulate files in your working copy on each test run.
You should mock the file system calls instead.